frontend/packages/console-app/src/components/pdb/PDBList.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { sortable } from '@patternfly/react-table';
+import * as classNames from 'classnames';
+import i18next from 'i18next';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { RowProps, TableColumn } from '@console/dynamic-plugin-sdk';
+import { useActiveColumns } from '@console/internal/components/factory/Table/active-columns-hook';
+import VirtualizedTable, {
+  TableData,
+} from '@console/internal/components/factory/Table/VirtualizedTable';
+import {
+  Kebab,
+  ResourceLink,
+  Timestamp,
+  ResourceKebab,
+  Selector,
+} from '@console/internal/components/utils';
+import { PodDisruptionBudgetModel } from '@console/internal/models';
+import { PodDisruptionBudgetKind, referenceForModel } from '@console/internal/module/k8s';
+
+const tableColumnInfo = [
+  { className: '', id: 'name' },
+  { className: '', id: 'namespace' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-sm'), id: 'selector' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-md'), id: 'minAvailable' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-lg'), id: 'disruptionsAllowed' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-xl'), id: 'creationTimestamp' },
+  { className: Kebab.columnClass, id: '' },
+];
+
+const { common } = Kebab.factory;
+const menuActions = [...Kebab.getExtensionsActionsForKind(PodDisruptionBudgetModel), ...common];
+
+const PodDisruptionBudgetTableRow: React.FC<RowProps<PodDisruptionBudgetKind>> = ({
```

Comment:
Can we split this out into its own component? Ideally we don't want to put so many different things together.

My rule of thumb here for our Tables is files like:
- List rendering
- Row rendering
- table configuration (for column info, table columns, etc)

It's easier to read imo and keeps the code clean and straight forward. The goal of separating files is when you're looking at `x` (list, details, etc, etc) you're not seeing the direct implementation of everything that made it work. You'll see imports and function calls where needed, and ideally they are descriptive enough that I don't need to know the column structure if I'm not trying to debug that aspect.